sale_timesheet_enhanced/models/account_analytic_line.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2018 Brainbean Apps
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models, api, _
+from odoo.exceptions import UserError
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = 'account.analytic.line'
+
+    exclude_from_sale_order = fields.Boolean(
+        string='Exclude from Sale Order',
+        help=(
```

Comment:
@alexey-pelykh IMO Here `help=(_('Checking this would exclude this timesheet entry from Sale Order'))`

Reply:
I haven't seen any usage (rgrep over OCA gives only 3 matches) of `_` with help - are there any need for that? @pedrobaeza I guess you're going to know the technical answer

Reply:
It's not needed. `help` string is automatically added to the translatable strings without the use of `_`. Even more, using it might waste translations. @nikul-serpentcs please be sure of what you ask in the reviews.